Demo/Configuration.swift

```diff
@@ -65,10 +65,7 @@ internal enum ConfigurationConstants {
     ]]
     
     static var delegatedAuthenticationConfigurations: ThreeDS2Component.Configuration.DelegatedAuthentication {
-        .init(localizedRegistrationReason: "Authenticate your card!",
-              localizedAuthenticationReason: "Register this device!",
-              appleTeamIdentifier: appleTeamIdentifier)
-        
+        .init(relyingPartyIdentifier: "test-authentication-adyen.netlify.app")
```

Comment:
Case in point: Creation of the config struct changed

Reply:
https://github.com/Adyen/adyen-ios/pull/1726#discussion_r1673530272